interface-definitions/interfaces_wireless.xml.in

```diff
@@ -528,19 +649,39 @@
               </valueHelp>
               <valueHelp>
                 <format>required</format>
-                <description>MFP enforced</description>
+                <description>MFP enforced (mandatory for WPA3)</description>
               </valueHelp>
               <constraint>
                 <regex>(disabled|optional|required)</regex>
               </constraint>
             </properties>
             <defaultValue>disabled</defaultValue>
           </leafNode>
+          <leafNode name="beacon-frame-protection">
```

Comment:
Boolean CLI nodes are discouraged. Please make this a `<valueLess/>` node instead and rename it to: `enable-bfp`.

Reply:
CR addressed in https://github.com/vyos/vyos-1x/pull/3524/commits/3619aa041c54c33b3b1b63694ca77fc9fc38badb